benchmarks/nel/requirements.txt

```diff
@@ -1,3 +1,7 @@
 pyyaml
+requests
```

Comment:
Is requests actually used anywhere?

Reply:
No, that was used for querying the Wiki API, which has been replaced by parsing the dump locally. Removed.